examples/color/colormaps_reference.py

```diff
@@ -57,7 +57,8 @@
                              'brg', 'CMRmap', 'cubehelix',
                              'gnuplot', 'gnuplot2', 'gist_ncar',
                              'nipy_spectral', 'jet', 'rainbow',
-                             'gist_rainbow', 'hsv', 'flag', 'prism'])]
+                             'gist_rainbow', 'hsv', 'flag', 'prism']),
+         ('Perceptually Uniform Cyclic', ['twilight', 'sunlight'])]
```

Comment:
Any reason the order was switched here from ` doc/users/plotting/colormaps/colormaps.py
`?